src/main/java/domain/Calculator.java

```diff
@@ -0,0 +1,14 @@
+package domain;
+
+import domain.dto.Operand;
+
+import java.util.List;
+
+public class Calculator {
+    public double calculate(List<String> operands) {
```

Comment:
```suggestion
    public long calculate(List<String> operands) {
```

mapToLong 하고 double 이길래 한번 넣어봐여 (suggestion 테스트)

Reply:
Thankyou <3